loadtest/control/actions.go

```diff
@@ -745,6 +745,136 @@ func Reload(u user.User) UserActionResponse {
 	return UserActionResponse{Info: "page reloaded"}
 }
 
+func ReloadGQL(u user.User) UserActionResponse {
```

Comment:
This is same as `control.Reload`, but with the REST calls removed which are covered in gql.

Reply:
Could you add this same comment as a comment in the function? It may not be 100% clear by reading the function's name only.